### PR TITLE
Flywheel base code

### PR DESCRIPTION
**Pull request by R7B9**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/flywheel/Flywheel.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    public Flywheel() {
+        fieldsTable.update();
+    }
+
+    @Override
+    public void periodic() {
+    }
+
+    public void setSpeed(double upperRollerDemandVoltage, double lowerRollerDemandVoltage) {
+        upperRollerDemandVoltage = upperRollerSpeedAccelerationLimiter.calculate(upperRollerDemandVoltage);
```
Comment (RBD120307) [inline src/main/java/frc/robot/subsystems/flywheel/Flywheel.java:34]: Why not making it two different functions and then calling them? Seems to me more elegant

File: src/main/java/frc/robot/subsystems/flywheel/FlywheelCommands.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import frc.robot.subsystems.flywheel.commands.RotateFlywheel;
+
+public class FlywheelCommands {
+    private final Flywheel flywheel;
+
+    public FlywheelCommands(Flywheel flywheel) {
+        this.flywheel = flywheel;
+    }
+
+    public Command rotate(DoubleSupplier upperRollerSpeedSupplier, DoubleSupplier lowerRollerSpeedSupplier) {
```
Comment (RBD120307) [inline src/main/java/frc/robot/subsystems/flywheel/FlywheelCommands.java:15]: rotate sounds weird. I would say something with "controller" is better

File: src/main/java/frc/robot/subsystems/flywheel/commands/RotateFlywheel.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    private final DoubleSupplier upperRollerSpeedDemandSupplierRPS;
+    private final DoubleSupplier lowerRollerSpeedDemandSupplierRPS;
+
+    private final PIDController upperRollerSpeedPID = new PIDController(KP, KI, KD);
+    private final PIDController lowerRollerSpeedPID = new PIDController(KP, KI, KD);
+
+    private final SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(KS, KV, KA);
+
+    public RotateFlywheel(
```
Comment (RBD120307) [inline src/main/java/frc/robot/subsystems/flywheel/commands/RotateFlywheel.java:23]: once again, weird name.

File: src/main/java/frc/robot/subsystems/flywheel/Flywheel.java (hunk 1 above)
Comment (R7B9) [reply src/main/java/frc/robot/subsystems/flywheel/Flywheel.java:34]: That what I did at the beginning, but then I was like "when will you call only one of them"? I don't know, makes it harder to forget to call both of them. Maybe will change

File: src/main/java/frc/robot/subsystems/flywheel/FlywheelCommands.java (hunk 2 above)
Comment (R7B9) [reply src/main/java/frc/robot/subsystems/flywheel/FlywheelCommands.java:15]: Yeah naming this was hard, and I don't like this name too, but `controller` seems even weirder to me. Think of when writing commands compositions, verbs that actually says what it does are more readable.

File: src/main/java/frc/robot/subsystems/flywheel/commands/RotateFlywheel.java (hunk 3 above)
Comment (R7B9) [reply src/main/java/frc/robot/subsystems/flywheel/commands/RotateFlywheel.java:23]: same response

**Pull request merged**